int32 input_case
---
int32 return
